sql/src/main/java/org/apache/druid/sql/calcite/schema/DruidSchema.java

```diff
@@ -107,6 +107,7 @@ public class DruidSchema extends AbstractSchema
   private final JoinableFactory joinableFactory;
   private final ExecutorService cacheExec;
   private final ExecutorService callbackExec;
+  private final ExecutorService metricsReporterExec;
```

Comment:
Oops - I will remove all these changes to this class. This was leftover from an earlier prototype.